content/en/news/2019/announcing-1.2-eol/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Support for Istio 1.2 ends on December 13th, 2019
+subtitle: Support Announcement
+description: Upcoming Istio 1.2 end of life announcement.
+publishdate: 2019-11-11
+aliases:
+    - /blog/2019/announcing-1.2-eol
+---
+
+According to Istio's [support policy](/about/release-cadence/), LTS releases like 1.2 are supported for three months after the next LTS release.   Since [1.3 was released on September 12th](/news/2019/announcing-1.3/), support for 1.2 will end on December 13th, 2019.
```

Comment:
```suggestion
According to Istio's [support policy](/about/release-cadence/), LTS releases like 1.2 are supported for three months after the next LTS release. Since [1.3 was released on September 12th](/news/2019/announcing-1.3/), support for 1.2 will end on December 13th, 2019.
```